terraform/environments/main.tf

```diff
@@ -5,7 +5,7 @@ variable "vault_addr" {
 module "aws_deploy-main-ap-southeast-1" {
   source            = "../modules/cloud/aws/deploy"
   env               = "main"
-  bootstrap_version = "stable"
+  bootstrap_version = "v2.0.1"
```

Comment:
I compared 4539fd4 (current stable branch) and v2.0.1.

v2.0.1 was taken from master, and seems more recent.

ok.